mass_mailing_switzerland/models/utm.py

```diff
@@ -204,6 +204,31 @@ class UtmCampaign(models.Model):
         compute="_compute_click_count", store=True, readonly=True
     )
 
+    def open_invoices(self):
```

Comment:
This would be a more appropriate name
```suggestion
    def open_analytic_lines(self):
```